hibernate-core/src/main/java/org/hibernate/annotations/SoftDelete.java

```diff
@@ -65,19 +64,6 @@
 	 */
 	String columnName() default "deleted";
```

Comment:
What do you think about defaulting to an empty string to choose between `deleted`/`active` based on `reverse`?

```suggestion
	String columnName() default "";
```

Reply:
I think that's a good idea.